lib/StyleCop/StyleCopCheck.proj

```diff
@@ -0,0 +1,34 @@
+<Project DefaultTargets="Full" xmlns= "http://schemas.microsoft.com/developer/msbuild/2003" ToolsVersion="4.0">
+
+    <UsingTask AssemblyFile="StyleCop.dll" TaskName="StyleCopTask"/>
+
+    <PropertyGroup>
+        <FolderToAnalyse>..\..\src</FolderToAnalyse>
+        <ProjectPath>..\..\src\chocolatey.csproj</ProjectPath>
+        <StyleCopForceFullAnalysis>true</StyleCopForceFullAnalysis>
+        <DefineConstants>DEBUG;TRACE</DefineConstants>
+        <StyleCopTreatErrorsAsWarnings>false</StyleCopTreatErrorsAsWarnings>
+        <StyleCopCacheResults>true</StyleCopCacheResults>
+        <StyleCopOverrideSettingsFile>src\Settings.StyleCop</StyleCopOverrideSettingsFile>
+        <StyleCopOutputFile>..\..\build_output\build_artifacts\StyleCop\StyleCopViolations.xml</StyleCopOutputFile>
+        <StyleCopMaxViolationCount>1000</StyleCopMaxViolationCount>
+    </PropertyGroup>
+    
+    <Target Name="Full">
+		<CreateItem Include="$(FolderToAnalyse)\**\*.cs">

```

Comment:
Yip, you can add an exclusion list for files that you don't want to be analyzed, that is not an issue.